membership.html

```diff
@@ -406,6 +421,19 @@ <h1 class="page-header">Join InternetNZ</h1>
                 </div>
 
             </form>
+
+            <div class="form-group row form-check text-right p-2">
+                <p>
+                    <input class="form-check-input"
+                           type="checkbox"
+                           name="generateExpired"
+                           id="generateExpired"
+                           value="true"
+                           v-model="userMembershipAttributes.generateExpired"
+                    >
+                    <label for="generateExpired" class="form-check-label policy-label font-light">Generate an expired membership</label>
```

Comment:
@camfindlay and @ehsan-fj I added a checkbox to enable an expired membership generation. Would be easy enough to add more use cases if we need them.